dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -1157,6 +1157,12 @@ private GHRef[] getBranchesAndTags(GHRepository repo) throws IOException {
             getBranchesSucceeded = true;
         } catch (GHFileNotFoundException ex) {
             LOG.debug("No branches found for " + repo.getName(), ex);
+        } catch (org.kohsuke.github.HttpException ex) {
+            if (ex.getResponseCode() == HttpStatus.SC_CONFLICT) {
+                // it seems like this is what is returned if there are absolutely no branches or tags
```

Comment:
Did some experiments and confirmed that a 409 (SC_CONFLICT) is returned when the repo is empty.  So weird, seems like the intuitive response would be success with an empty result, as is the case when the repo is not empty but no refs are matched.